### PR TITLE
removed arrays code, renamed packages

### PR DESCRIPTION
**Pull request by kcibul**
Conceptually -- two things happen in this PR. 

1. Removed "arrays" code
2. Refactored package names into

...gvs
     |- ingest
     |- filtering
     |- extract
     \- common

All tests pass (can be run with `./gradlew test --tests "org.broadinstitute.hellbender.tools.gvs*"`).  I will run a full workflow in Terra (import, train, extract) and verify

BEFORE we merge, we'll the existing "ah_var_store" to indicate this is where array code lives

### CODE REVIEW

**Comment by kcibul** [comment]
On second thought -- to simplify the merge to master we may take a different approach (copying files, squashing, etc) so I'm updating the approach above for this PR

**Review by RoriCremer — APPROVED** [review]

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java [hunk 1]
```diff
@@ -178,8 +178,6 @@ public void onTraversalStart() {
             case GENOMES:
                 vetTsvCreator = new VetTsvCreator(sampleIdentifierForOutputFileName, sampleId, tableNumberPrefix, outputDir);
                 break;
-            case ARRAYS:
```
Comment (ahaessly) [inline src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java:181]: do we even need this switch statement? we make the same call for both exomes and genomes

**Review by ahaessly — APPROVED** [review]
one minor suggestion

File: src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java (hunk 1 above)
Comment (kcibul) [reply src/main/java/org/broadinstitute/hellbender/tools/gvs/ingest/CreateVariantIngestFiles.java:181]: Yes!  I was restraining myself to clean up code and save that for the next PR.  Mostly to make the review of this PR easy (ie it's all structural, you don't need to look at each line of code) and the next one smaller, but where the lines of code matter.  We also have some "common" code that is now really tool specific that we can shift.  Things like the above, etc.

**Pull request merged**